src/Main.java

```diff
@@ -1,6 +1,89 @@
 
 public class Main {
     public static void main(String[] args) {
-        System.out.println(" Coursework 1");
+        System.out.println("Coursework 1");
+        Employee[] people = new Employee[10];
+        int idCurrentEmployee = -1;
+        AutoIncrementCounter id = new AutoIncrementCounter();
+
+        people[0] = new Employee (id.getNextId(), "Иван", "Иванов", "Иванович", 1,120000);
+        people[1] = new Employee (id.getNextId(), "Петр", "Петров", "Петрович", 1,110000);
+        people[2] = new Employee (id.getNextId(), "Денис", "Денисов", "Денисович", 2,130000);
+        printCompleteDateOfEmployees(people);
+
+        int summa = getCostAmount(people);
+        System.out.println("Сумма затрат в месяц равна " + summa);
+        idCurrentEmployee = minSumma(people);
+        System.out.println("Сотрудник с минимальной зарплатой " + people[idCurrentEmployee]);
+        idCurrentEmployee = maxSumma(people);
+        System.out.println("Сотрудник с максимальной зарплатой " + people[idCurrentEmployee]);
+        summa = averageSumma(people);
+        System.out.println("Средняя зарплата сотрудников: "+ summa);
+        System.out.println("Список ФИО сотрудников:" );
+        printListEmployees(people);
+    }
+
+    public static int getCostAmount(Employee[] employees) {
+        int summa = 0;
+        for (int i = 0; i < employees.length; i++) {
+            if (employees[i] != null) {
+                summa += employees[i].getSalary();
+
+            }
+        }
+        return summa;
+    }
+
+    public static int minSumma(Employee[] employees) {
+        int minSumma = 10000000;
```

Comment:
почему не 20 млн? Метод будет работать неверно, если у какого-то сотрудника будет ЗП больше. Здесь надо использовать максимально возможное число для int
```suggestion
        int minSumma = Integer.MAX_VALUE;
```